scormxblock/scormxblock.py

```diff
@@ -495,9 +495,13 @@ def _publish_grade(self, status, score):
 
     def publish_progress(self, scorm_data):
         """
-        Mark 100% completion if course is complete
+        Update progress % if cmi.progress_measure is emitted
+        Else check status and mark 100% completion if course is complete
         """
-        if scorm_data.get('status', '') in constants.SCORM_COMPLETION_STATUS:
+        progress_measure = self.calc_progress_measure(scorm_data)
+        if progress_measure > 0:
```

Comment:
No need for > 0. if progress_measure will work.

Reply:
Please read the doc string.

Reply:
I mean 
if progress_measure:
this will work.

Reply:
oh, Sure! I misunderstood